src/translate/invariant_finder.py

```diff
@@ -126,6 +126,7 @@ def useful_groups(invariants, initial_facts):
     for (invariant, parameters) in useful_groups:
         yield [part.instantiate(parameters) for part in sorted(invariant.parts)]
 
+# returns a list of mutex groups (parameters instantiated, counted variables not)
```

Comment:
I added that the return type is a list[list[Atom]], but kept the comment because it contains the additional information to what extent the atoms are grounded.